template_overlay_typescript_components/src/app.tsx

```diff
@@ -25,7 +25,7 @@ export default class MyApp extends React.Component<Props, State> {
     return (
       <View name="main-view">
         <Text textSize={0.1} localPosition={[-0.3, 0, 0]}>
-          {this.state.message}
+          Hello Magic Script
```

Comment:
Since this removes all use of state, we can drop the State interface above and all the lines Pawel mentioned.